recipes/nolearn/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "a7779c7d7d4bcf34d4db439df249eadbba04f3dd09ee7579f07ddf24e5bfc579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy==1.10.4
+    - scipy==0.16.1
+    - scikit-learn==0.17
+    - joblib==0.9.3
+    - Theano==0.8
+    
+  run:
+    - python
+    - numpy==1.10.4
```

Comment:
I believe conda-forge only supports >=1.11. Are all of these hard-pinning of versions required? I can see them listed in the `requirements.txt`, but it's unclear if they are needed.